### DISCUSSION TITLE
Create .lib file from .dll (archive) | Valentin Popov

### DISCUSSION THREAD

**Discussion by giscus** [valentineus/popov.link#1] [Announcements]
# Create .lib file from .dll (archive) | Valentin Popov

Quick guide to create a .lib from a .dll on Windows: list exports with dumpbin, make a .def file, then generate the import library with lib.

https://popov.link/blog/create-lib-file-from-dll/

**Comment by valentineus** [comment]
I express my gratitude to the author of the original article — __Adrian Henke__ 🎉.
